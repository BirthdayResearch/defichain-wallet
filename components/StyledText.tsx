import * as React from 'react'

import { Text, TextProps } from './Themed'

export function MonoText (props: TextProps): JSX.Element {
  return <Text {...props} style={[props.style, { fontFamily: 'space-mono' }]} />
}
