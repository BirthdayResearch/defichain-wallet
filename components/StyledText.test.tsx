import * as React from 'react'
import renderer from 'react-test-renderer'

import { MonoText } from './StyledText'

it('<MonoText/> snapshot', () => {
  const tree = renderer.create(<MonoText>Snapshot test!</MonoText>).toJSON()

  expect(tree).toMatchSnapshot()
})
