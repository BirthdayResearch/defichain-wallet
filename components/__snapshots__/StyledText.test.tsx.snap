// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonoText/> snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Array [
        undefined,
        Object {
          "fontFamily": "space-mono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
